docs/cugraph/source/tutorials/basic_cugraph.md

```diff
@@ -5,7 +5,7 @@
 CuGraph is part of [Rapids](https://docs.rapids.ai/user-guide) and has the following system requirements:
  * NVIDIA GPU, Volta architecture or later, with [compute capability](https://developer.nvidia.com/cuda-gpus) 7.0+
  * CUDA 11.2, 11.4, 11.5, 11.8, 12.0 or 12.2
```

Comment:
We might want to slip this update in here too.
```suggestion
 * CUDA 11.2, 11.4, 11.5, 11.8, 12.0, 12.2, or 12.5
```

Reply:
sure! fine with me